### PR TITLE
Improve the look of page components

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: src/main/resources/site/pages/sample-page/sample-page.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<h3>Page Components</h3>
+<p>Components that you added to the page will be shown here:</p>
+<main th:if="${regions.main.components} and ${#lists.size(regions.main.components) > 0}" th:data-portal-region="main" class="component-page">
+	<h4>Sample Page</h4>
+	<div th:each="component : ${regions.main.components}" th:remove="tag">
+		<div data-portal-component="${component.path}" th:remove="tag"/>
+	</div>
+</main>
+
+<h3>Localisation</h3>
```
Comment (ComLock) [inline src/main/resources/site/pages/sample-page/sample-page.html:25]: Localization?

**Review by ComLock — CHANGES_REQUESTED** [review]
I'm unable to add anything to the page, seems like regions are not working?

File: src/main/resources/site/pages/sample-page/sample-page.html (hunk 1 above)
Comment (alansemenov) [reply src/main/resources/site/pages/sample-page/sample-page.html]: will be american english, but fine.

**Comment by alansemenov** [comment]
> I'm unable to add anything to the page, seems like regions are not working?

Fixed

**Review by ComLock — APPROVED** [review]

**Pull request merged**